R/tmb.R

```diff
@@ -129,7 +129,12 @@ h_mmrm_tmb_data <- function(formula_parts,
   # weights is always the last column
   weights_name <- colnames(data)[ncol(data)]
   full_frame <- eval(
```

Comment:
I wonder if we should raise a warning if the global option is set differently than na.omit?

Reply:
it is good idea to pop up the error here; however, this can be quite a large update; I can open new issues to address. The reason that I say this is quite large because currently `h_mmrm_tmb_data` is called multiple times in different fit (it should be done only once as the data should be the same, but the optimizer changes). If the warning messages are created, then in every fit try, the warnings are captured by `h_record_all_output` and only when number of warnings is 0 we say this is converged/ or we handle that warning so that make the converge but this message will be discarded in `h_record_all_output` just like "NA/NAN function evaluation" . So having a warning here will not display the message directly, but difficult to handle. We need to separate the "h_mmrm_tmb_data` and `fit_single_optimizer`, but this will make `fit_single_optimizer` a big breaking change (it could not be called directly, it need to be feeded with tmb_data) and I doubt if it is possible

Reply:
instead, we can check the options in `mmrm` call, or in `fit_single_optimizer` call. But this is really not the best place to give the warning message because it is in`h_mmrm_tmb_data` the `na.action` is used and checked.

Reply:
oh probably message can be used. let me check...

Reply:
ok that would be good then